### ISSUE TITLE
Weird error when fuseing my existing repo.

### ISSUE THREAD

**Issue by lucasew** [borgbackup/borg#4307]
Please fill in as much of the template as possible.
-->

## Have you checked borgbackup docs, FAQ, and open Github issues?

Yes

## Is this a BUG / ISSUE report or a QUESTION?

Bug

#### Your borg version (borg -V).
Tested it in borg 1.1.8 and 1.1.7

#### Operating system (distribution) and version.
Arch, latest.

#### Hardware / network configuration, and filesystems used.
I5 7200U
HD for storage, the repo filesystem is a common NTFS partition.

#### How much data is handled by borg?
The repo is 117.4GB now.

#### Full borg commandline that lead to the problem (leave away excludes and passwords)
Im letting asciinema casts for it.

## Describe the problem you're observing.
I create a dir in /tmp called borg, use borg mount to mount the repo here, I can ls in the first level of folders without problems but if I ls inside of one of this folders borg closes with a error.

#### Can you reproduce the problem? If so, describe how. If not, describe troubleshooting steps you took before opening the issue.
Its in the asciinema casts

#### Include any warning/errors/backtraces from the system logs

```
➜  /tmp borgfs -f --info /media/dados/Lucas/BACKUP/borg borg
Enter passphrase for key /media/dados/Lucas/BACKUP/borg: 
Mounting filesystem
handler raised <class 'TypeError'> exception (unpack() takes no arguments (1 given)), terminating main loop.
Local Exception
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/borg/archiver.py", line 4434, in main
    exit_code = archiver.run(args)
  File "/usr/lib/python3.7/site-packages/borg/archiver.py", line 4366, in run
    return set_ec(func(args))
  File "/usr/lib/python3.7/site-packages/borg/archiver.py", line 1352, in do_mount
    return self._do_mount(args)
  File "/usr/lib/python3.7/site-packages/borg/archiver.py", line 152, in wrapper
    return method(self, args, repository=repository, **kwargs)
  File "/usr/lib/python3.7/site-packages/borg/archiver.py", line 1362, in _do_mount
    operations.mount(args.mountpoint, args.options, args.foreground)
  File "/usr/lib/python3.7/site-packages/borg/fuse.py", line 340, in mount
    signal = fuse_main()
  File "/usr/lib/python3.7/site-packages/borg/fuse.py", line 34, in fuse_main
    return llfuse.main(workers=1)
  File "src/fuse_api.pxi", line 330, in llfuse.main
  File "src/handlers.pxi", line 436, in llfuse.fuse_opendir
  File "src/handlers.pxi", line 437, in llfuse.fuse_opendir
  File "/usr/lib/python3.7/site-packages/borg/fuse.py", line 581, in opendir
    self._load_pending_archive(inode)
  File "/usr/lib/python3.7/site-packages/borg/fuse.py", line 554, in _load_pending_archive
    self.process_archive(archive_name, [os.fsencode(archive_name)])
  File "/usr/lib/python3.7/site-packages/borg/fuse.py", line 377, in process_archive
    consider_part_files=self.args.consider_part_files):
  File "/usr/lib/python3.7/site-packages/borg/fuse.py", line 160, in iter_archive_items
    item = unpacker.unpack(write_bytes)
TypeError: unpack() takes no arguments (1 given)

Platform: Linux acer-arch 4.20.5-arch1-1-ARCH #1 SMP PREEMPT Sat Jan 26 12:59:18 UTC 2019 x86_64
Linux: arch Arch Linux
Borg: 1.1.7  Python: CPython 3.7.2
PID: 3102  CWD: /tmp
sys.argv: ['/usr/bin/borgfs', '-f', '--info', '/media/dados/Lucas/BACKUP/borg', 'borg']
SSH_ORIGINAL_COMMAND: None
```

[casts.zip](https://github.com/borgbackup/borg/files/2824581/casts.zip)

**Comment by ThomasWaldmann** [comment]
see #4245.

**Comment by lucasew** [comment]
Workarounded with a virtualenv and changing the shebang of borg and borgfs pointing to virtualenv's python. The problem seems to be newer msgpack not supported.

**Comment by ThomasWaldmann** [comment]
@lucasew isn't it enough to just call the "borg" (or "borgfs") in the virtualenv_dir/bin/ directory?

**Comment by lucasew** [comment]
I have created the virtualenv with pyenv that I have already installed.
I think if msgpack 0.5.6 was on arch repos I could not need this.
Then I installed borg and its dependencies (msgpack=0.5.6 borgbackup and llfuse) via pip in a virtualenv I have created it with pyenv that I already have.
After installing that three packages via pip I have edited the executables: /usr/bin/borg and /usr/bin/borgfs. I just edited the shebang pointing to the python inside the virtualenv and it works.

![2019-02-02_18 10 25_1366x768_scrot](https://user-images.githubusercontent.com/15693688/52168805-def67100-2715-11e9-842f-0887a00f7425.png)
